protocol-units/execution/suzuka/executor/src/lib.rs

```diff
@@ -24,10 +23,9 @@ pub trait SuzukaExecutor {
     /// Runs the necessary background tasks.
     async fn run_background_tasks(&self) -> Result<(), anyhow::Error>;
 
-    /// Executes a block dynamically
-    async fn execute_block(
+    /// Executes a block optimistically
+    async fn execute_block_opt(
```

Comment:
Not sure if this is a good name, make your suggestions.

Reply:
Yeah, that's good.